ckanext/qgov/common/authenticator.py

```diff
@@ -34,18 +35,16 @@ def unlock_account(account_id):
 def intercept_authenticator():
     """ Replaces the existing authenticate function with our custom one.
     """
-    QGOVAuthenticator().orig_authenticate = UsernamePasswordAuthenticator.authenticate
-    UsernamePasswordAuthenticator.authenticate = QGOVAuthenticator().authenticate
+    # QGOVAuthenticator().orig_authenticate = UsernamePasswordAuthenticator.authenticate
+    # UsernamePasswordAuthenticator.authenticate = QGOVAuthenticator().authenticate
```

Comment:
If we comment out both of these, what will cause our authenticator to actually be used?